### ISSUE TITLE
[Request] Add feature to regain customizable number of Hit Dice on a Short Rest

### ISSUE THREAD

**Issue by leftyt13** [fantasycalendar/FoundryVTT-RestRecovery#139]
Hello. 

I'd like to request that an option to regain Hit Dice on a Short Rest be added. I'm planning on running some alternative rest rules for a game, and I came upon your module while looking for a way to make Foundry takes care of it for my players. _(I'll explain the rest rules below in case that's helpful, but will spare you that now in case it's not, haha)._ Ideally, one would get all the same options as Long Rests... control over how many Hit Dice you regain, whether that happens at the start or end of the rest, the buffer, all that good stuff.

Here's the rest rules I'd like to run, if it helps to know what my end goal is:
"Quick Rest" - 10 minutes, you can spend Hit Dice to heal.
"Short Rest" - 8 hours, you regain a number of Hit Dice equal to your Proficiency Bonus, you can roll Hit Dice to heal if needed, SR-based abilities/items/etc. recharge.
"Long Rest" - 24 hours, must be in civilization, you regain all Hit Dice, you regain all HP, LR-based abilities/items/etc. recharge.

Your module lets me do everything I need to make my alternative Long Rests work, so I'm golden there (thank you!), and Quick Rests are so easy that I don't feel the need for a module to facilitate them. So basically the only thing I can't make happen right now is the regaining of Hit Dice on Short Rests - if your module could do that also, I'd be totally set!

**TL;DR:** Option to regain Hit Dice on Short Rest please?

Thanks for your hard work on the module, and for taking the time to consider my request.

-Lefty
